README.md

```diff
@@ -315,19 +314,32 @@ Modifiers are subclasses, or variants, of a component. They add a suffix to the
 
 When extending a component and styling the inner elements, use the base component's inner elements' class name for styling, instead of extending the class names of the inner elements as well.
 
+
 ```css
 
 /* Good - modifiers (subclasses) of components refer to unmodified inner element's names */
-.c-list-inline--boxy > .o-list-inline__list-item {}
+.c-assignment-list--boxy > .c-assignment-list__item {}
 
 /* Bad - don't modify inner element's names */
-.c-list-inline--boxy > .c-list-inline__list-item--boxy {}
+.c-assignment-list--boxy > .c-assignment-list__list-item--boxy {}
 
 /* Bad - don't create child elements with the modifier in the name. */
-.c-list-inline--boxy__list-item {}
+.c-assignment-list--boxy__list-item {}
 ```
 
 
+<a name="feature-specific-styles"></a>
+### Feature-specific styles
+
+If it is uncertain whether a new class belongs in the library (i.e., will be re-used, and NEVER changed), then it should be a feature class.
+
+* Feature styles go in stylesheets/music/features. For any feature that doesn't yet exist as a subfolder, add one for your styles. 
+
+* Feature stylesheets should NOT be included in the main stylesheet; instead, add a stylesheet `<link>` (or a Rails `stylesheet_link_tag`) only to the views that need them.
```

Comment:
For some reason I remembered that as "no namespaces, just separate files."  

I can't think of a reason why namespaces and one big file would be a problem.